Convert textile to markdown.

hiroyuki-sato opened the pull request:
Following commands executed.

```
  for i in *.textile ; do
    f=$( basename $i .textile )
    pandoc $i -o /tmp/mdtest/${f}.md
  done

  cd /tmp/mdtest
  perl -i -pe 's/\\//g' *.md

  for i in *.md ; do gsed -i '2d' $i ;done
  for i in *.md ; do gsed -i '1c---' $i ;done

  cd ${GROONGA_ORG}/en/_posts

  for i in *.textile ; do
    f=$( basename $i .textile )
    git mv $i ${f}.md
  done

  mv /tmp/mdtest/*.md .
```

Commit:
Convert textile to markdown.

Following commands executed.

  for i in *.textile ; do
    f=$( basename $i .textile )
    pandoc $i -o /tmp/mdtest/${f}.md
  done

  cd /tmp/mdtest
  perl -i -pe 's/\\//g' *.md

  for i in *.md ; do gsed -i '2d' $i ;done
  for i in *.md ; do gsed -i '1c---' $i ;done

  cd ${GROONGA_ORG}/en/_posts

  for i in *.textile ; do
    f=$( basename $i .textile )
    git mv $i ${f}.md
  done

  mv /tmp/mdtest/*.md .

Commit:
Convert textile to markdown in ja/_posts directory.

kenhys commented:
PRを確認してみました。

これjekyll serve --watchでプレビューしようとするとエラーでません？
description: のところで意図しない改行をしているせいでエラーになるとか、{% post_url %}を使っているところがsyntax errorになってました。エラーを抜粋するとこんな感じでした。

```
             Error: could not read file /home/kenhys/work/groonga/groonga.org/en/_posts/2013-12-16-drop-squeeze-and-lucid-support.md: (<unknown>): could not find expected ':' while scanning a simple key at line 4 column 1
  Liquid Exception: Liquid syntax error: Tag '{%) post_url 2012-03-27-logo %}' was not properly terminated with regexp: /\%\}/ in /home/kenhys/work/groonga/groonga.org/ja/_posts/2012-03-29-release.md
```

そこをなおしてもらえると助かります。

Commit:
Remove unnecessary newline

Commit:
Convert h2. to ##

Commit:
Fix liquid markup error.

Fix following error.
  Tag '{%) post_url 2012-10-10-groonga-night-3 %}'

perl -i -pe 's/{%\)/{%/g' *.md

hiroyuki-sato commented:
失礼しました。`jekyll server`で確認すればよかったんですね。
手元の環境ではとおるようにしてみました。再度確認をお願いします。

kenhys commented:
それなりに変換対象のものが残っていたのでとても助かりました。
ありがとうございます！

いくつか変なところはあるのですが、ささいな変更なのでそれはなおしておきます。

hiroyuki-sato commented:
ありがとうございました。
